### PR TITLE
Setup Jaeger tracing for glusterd2 volume create command.

### PR DESCRIPTION
**Pull request by sseshasa**
This change sets up Jaeger tracing for volume create path. This initial changeset only traces calls locally and not across rpc boundaries. Therefore, the traces are only visible on the node which handled the volume create request.

None of the code changes modifies the original functionality and therefore UTs and functional tests are not added. But all existing UTs, functional tests and testing on VM with 1 node succeeded.

github issue #941 tracks this change.

### CODE REVIEW

**Comment by sseshasa** [comment]
Unable to add reviewers. Request @aravindavk, @prashanthpai, @kshlm to review the changes and add other folks as well.

File: glusterd2/main.go [hunk 1]
```diff
@@ -25,6 +25,9 @@ import (
 	config "github.com/spf13/viper"
 	"github.com/thejerf/suture"
 	"golang.org/x/sys/unix"
+
```
Comment (rishubhjain) [inline glusterd2/main.go]: I guess this extra line is not required since these are 3rd party libraries

Comment (sseshasa) [reply glusterd2/main.go]: Noted. Will remove it.

**Comment by Madhu-1** [comment]
do we need to update Gopkg.toml file?

**Comment by aravindavk** [comment]
> @Madhu-1 The Gopkg.toml file didn't need any change. Only the Gopkg.lock was autogenerated and which has been included in the first couple of commits.

New dependency details to be updated in `Gopkg.toml` file

**Comment by sseshasa** [comment]
@aravindavk I will update the Gopkg.toml with the new dependencies.

**Comment by prashanthpai** [comment]
add to whitelist

**Comment by sseshasa** [comment]
I am closing this PR as I messed up the git history for the associated fork. I will open a new PR with a different branch.

**Pull request closed without merging**